modules/ROOT/pages/dataweave-cookbook-extract-data.adoc

```diff
@@ -4,276 +4,177 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: studio, anypoint, transform, transformer, metadata, dataweave, data weave, datawave, datamapper, dwl, dfl, dw, payload, attribute, variable, flow, error, message
 
-This DataWeave example extracts a component of a Mule event.
-
-Extracted values are handled as a literal value (as opposed to a variable, for example) of one of the supported DataWeave value types.
-
-[cols="1,1,1", options="header"]
-|===
-| Data to extract | Anypoint Studio syntax | Design Center syntax
-
-| *Payload*
-| `#[payload]`
-| `payload`
-
-| *Attribute*
-| `#[attributes.<myAttributeName>]`
-| `attributes.<myAttributeName>`
-
-| *Variable*
-
-To avoid name collisions, you can prepend `variables`, as shown.
-| `\#[<myVariableName>]`
-or
-`#[variables.<myVariableName>]`
-| `<myVariableName>`
-or
-`variables.<myVariableName>`
-
-| *Error object*
-
-For information on errors in the flow, you can use `#[error.cause]`.
-| `#[error]`
-| `error`
-
-| *Flow*
-| `\#[flow]`
-
-For the flow name: `#[flow.name]`
-| `flow`
-|===
+DataWeave can select data from objects, arrays, and variables that store data
```

Comment:
Back to this sentence - can we just say "DataWeave can select data from objects, arrays, and variables